packages/processors/src/strategy/helpers/applicationStatus.ts

```diff
@@ -0,0 +1,47 @@
+import { Application } from "@grants-stack-indexer/repository";
+
+/**
+ * Checks if an application status index is valid (between 1 and 3)
+ * @see ApplicationStatus
+ */
+export function isValidApplicationStatus(status: number): boolean {
+    return status >= 1 && status <= 3;
+}
```

Comment:
Don't like this hardcoding that much. But it is something that probably won't change. 

What about using `Object.keys(ApplicationStatus).length` ?

Reply:
didn't follow exactly your idea, you mean to use the length as the upper bound limit?